app/serializers.py

```diff
@@ -118,15 +130,53 @@ def to_representation(self, instance):
 
 class EmployeeSerializer(serializers.ModelSerializer):
     user = EmployeeUserSerializer()
-    degrees = DegreeSerializer(many=True, read_only=True)
-    experience = ExperirenceSerializer(many=True, read_only=True)
+    degrees = DegreeSerializer(many=True)
+    assets = AssetSerializer(many=True)
+    experience = ExperirenceSerializer(many=True)
     org_id = serializers.CharField(read_only=True)
-
+    managing = serializers.PrimaryKeyRelatedField(
+        write_only=True, queryset=models.Employee.objects.all(), many=True
+    )
     total_experience = serializers.SerializerMethodField()
+    manages = serializers.StringRelatedField(read_only=True, many=True)
 
     class Meta:
         model = models.Employee
-        exclude = ("organization",)
+        exclude = ("organization", "slack_id")
+
+    @transaction.atomic
+    def create(self, validated_data):
+        user_data = validated_data.pop("user")
+        degrees_data = validated_data.pop("degrees")
+        experience_data = validated_data.pop("experience")
+        assets_data = validated_data.pop("assets")
+        managing = validated_data.pop("managing")
+        if user_data.get("default_role"):
+            user_data["default_role"] = user_data.pop("default_role").id
+        organization = self.context.get("request").user.organization
+        user_ser = EmployeeUserSerializer(data=user_data)
+        user_ser.is_valid(raise_exception=True)
+        user = user_ser.save()
+        user.username = user.email
+        user.organization = organization
+        user.save()
+        validated_data["user_id"] = user.id
+        employee = super().create(validated_data)
+
+        for manages in managing:
+            emp = models.Employee.objects.get(id=manages.id)
```

Comment:
models.Employee.objects.get(id=manages.id).update(field2='cool')

We can do it in one line

Reply:
also convert loops to generators